asteroid/metrics.py

```diff
@@ -258,5 +295,8 @@ def final_df(self):
         )
         return df
 
+    def all_transcriptions(self):
+        return dict(transcriptions=self.transcriptions)
```

Comment:
I don't really see the point of the dict with one field, returning the list. 
I'd remove this method entirely